radio/src/Makefile

```diff
@@ -850,6 +856,8 @@ ifeq ($(PCB), $(filter $(PCB), SKY9X 9XRPRO AR9X))
   ifeq ($(RTCLOCK), YES)
     CPPDEFS += -DRTCLOCK
     CPPSRC += rtc.cpp targets/sky9x/rtc_driver.cpp
+  else ifeq ($(PCB), SKY9X)
+    CPPSRC += targets/sky9x/rtc_driver.cpp
```

Comment:
I would not add this but rather complete the `#if defined(REVX)` you added for this PR by `#if defined(REVX) && defined(RTCLOCK)`